apps/src/code-studio/initApp/loadApp.js

```diff
@@ -302,7 +306,8 @@ function loadAppAsync(appOptions) {
         `/${appOptions.scriptName}` +
         `/${appOptions.lessonPosition}` +
         `/${appOptions.levelPosition}` +
-        `/${appOptions.serverLevelId}`
+        `/${appOptions.serverLevelId}` +
+        `?load_channel=${shouldLoadChannel}`
```

Comment:
The name `load_channel` is a little confusing to me because I think we use it elsewhere to mean retrieving the channel data.  Maybe something like `get_channel_id` instead for the query param name?